package.json

```diff
@@ -26,16 +26,16 @@
     "lockdrop"
   ],
   "dependencies": {
-    "yargs": "^13.0",
-    "web3": "^1.2",
-    "chai": "^4.2.0",
     "bn-chai": "^1.0.1",
+    "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "eth-gas-reporter": "^0.1.12",
-    "ganache-cli": "6.4.2",
+    "ganache-cli": "^6.9.1",
```

Comment:
Maybe, the CI error's cause is `ganahce-cli` version.
This should be `"ganache-cli": "^6.4.2",`.
And I think this repository should include `package-lock.json` because of fixing the library version.